src/go.mod

```diff
@@ -1,76 +1,79 @@
 module code.cloudfoundry.org/log-cache
 
-go 1.21
+go 1.21.0
+
+toolchain go1.22.0
```

Comment:
I don't think that adding this line is a good idea. It may result in development of this module using go1.22.0, while we the release is actually built using go1.21.0.

Reply:
https://go.dev/doc/toolchain